airbyte-webapp/src/hooks/services/useConnector.test.tsx

```diff
@@ -1,73 +0,0 @@
-import { act, renderHook } from "@testing-library/react-hooks";
```

Comment:
The testing could likely be rewritten, but the method signatures are really light comparatively.